web-console/future-openshift/command-line-tools/command-line-tools.md

```diff
@@ -0,0 +1,19 @@
+# Command Line Tools
+
+If users do not already have the `oc` command line tool, they can download the version corresponding to the OpenShift cluster they are using from the Command Line Tools page. They can also download OpenShift do (`odo`) from this page. More command line tools will be added to this page over time.
+
+## Help Menu
+![dropdown](img/dropdown.png)
+* The command line tools page will be accessible from the help menu.
+
+## Modal
+![modal](img/modal.png)
+* Currently, users can only download `oc` or `odo`. For the time being, the command line tools will be displayed in a modal, with descriptions and download links for each tool.
+
+## Card view
+![card](img/card-view.png)
+* When more tools are added, the command line tools page will be a full page card view. Each card will have the tool name, associated logo, and description.
```

Comment:
Maybe worth noting the descriptions are placeholder and need to be reviewed if/when we add these other options in the future.